_content/essays/in-praise-of-tso-pema_chokyi-lodro.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "In Praise of Tso Pema"
+authors:
+  - chokyi-lodro
+translator: "Adam Pearcey"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/jamyang-khyentse-chokyi-lodro/tso-pema-praise"
+drive_links:
+  - "https://drive.google.com/file/d/1GLkm_m4m2F-X0qpugWgj05P4LJIWqNoN/view?usp=sharing"
+course: tibetan
+tags:
+  - mythology
+year: 2018
+publisher: "lotsawa house"
```

Comment:
I know we're going to clean these up soon, but normally these need Title Case (unless the publisher really insists on lower case, like punctum books)